### DISCUSSION TITLE
Save only annotations

### DISCUSSION THREAD

**Discussion by g-kartik** [gildas-lormeau/SingleFile#1409] [Ideas]
Is there a way to create and save just the annotations without generating a HTML copy of the website? 

When this can be useful:
- When user repeatedly visits a same website and he wants to go see the annotations he made earlier.
- This can be mostly useful for static webpages like Wikipedia, software documentation which we repeatedly visit.
- HTML pages can use lot of storage space over time as compared to just saving the annotations. This can be a concern when taking cloud backups.

Idea:
- User annotates stuff on a webpage. They may include notes, highlights etc. 
- These annotations are saved locally on users machine at some user-defined destination
- When user visits the same website, the annotations are loaded from the local storage and displayed over the webpage.
- We can show a sidebar containing list of all annotations made on the webpage and upon clicking user can navigate to them 
- Optionally the annotations can be backed up to cloud storage.

**Comment by gildas-lormeau** [comment]
I agree that it would be useful, but it's outside the scope of the extension. The main purpose of SingleFile is to save pages and I can't remove this functionality.
